Add git workflow instructions to readme

aaronsn opened the pull request:
- Describe forking workflow
- Suggest using "Squash and merge"
- Explain downsides to "Squash and merge" and how to do dependent changes.

Commit:
Add git workflow instructions to readme

Commit:
update link and bullet spacing

Commit:
minor fix

Commit:
Fix readme formatting

Commit:
Fix more formatting

Commit:
drop unnecessary part of the readme

jenniebrown commented on README.md:
```diff
@@ -1,7 +1,37 @@
-## One-time setup
+# Heath Equity Tracker
+
+The main codebase for [Health Equity Tracker](https://healthequitytracker.org/).
```
This is actually the prototype and not the main codebase for HET. I'd leave this line out

jenniebrown commented on README.md:
```diff
@@ -1,7 +1,37 @@
+
+## Contributing
+To contribute to this project:
+1. Fork the repository on github.
```
Maybe it would be helpful to link to this page here? https://docs.github.com/en/free-pro-team@latest/github/getting-started-with-github/fork-a-repo

jenniebrown reviewed (changes requested):
Thanks for writing this up! I think this will be helpful for our group. I might suggest for the main repository that we have a separate contributing.md file that is more of a "house rules" document instead of an abridged forking workflow tutorial.

Commit:
Merge from master

aaronsn commented:
Sorry the merge got messed up when I tried to pull in your readme changes. Made a new PR here: https://github.com/SatcherInstitute/prototype/pull/50 and addressed your comments there
